client/service-worker.js

```diff
@@ -0,0 +1,22 @@
+var cacheName = "notes-cache-v1";
```

Comment:
Can we not move this into public directory?

Reply:
Why not?